Modules/_interpchannelsmodule.c

```diff
@@ -11,7 +11,9 @@
 #include "pycore_pystate.h"       // _PyInterpreterState_GetIDObject()
 
 #ifdef MS_WINDOWS
+#ifndef WIN32_LEAN_AND_MEAN
 #define WIN32_LEAN_AND_MEAN
```

Comment:
nitpick: can you add two spaces for readability? `#  define WIN32_LEAN_AND_MEAN`.